crypto/aes/asm/aes-riscv64.pl

```diff
@@ -1066,7 +1066,8 @@ sub do_dec_lookup {
     addi    sp,sp,-16
     sd      $KEYP,0(sp) # We need to hold onto this!
     sd      ra,8(sp)
-    jal     ra,AES_set_encrypt_key
+    la      t0,AES_set_encrypt_key
+    jalr    ra,t0
```

Comment:
try `call AES_set_encrypt_key` instead,
so the linker will detect if a temporary register is necessary,
based on the address distance.